server/fleet/software.go

```diff
@@ -86,3 +93,31 @@ type SoftwareListOptions struct {
 	// a count of hosts > 0.
 	WithHostCounts bool
 }
+
+// SoftwareVulnerability identifies a vulnerability on a specific software (CPE).
+type SoftwareVulnerability struct {
+	ID    uint   `db:"software_id"`
```

Comment:
Consider renaming to `SoftwareID` (`ID` could be confused with the ID of the "Vulnerability").